tests/test_ikala.py

```diff
@@ -1,45 +1,118 @@
 from __future__ import absolute_import
 
-import os
-import json
+import numpy as np
 
 import pytest
 
 from mirdata import ikala, utils
-from tests.test_utils import mock_validated, mock_validator
 
 
-@pytest.fixture
-def mock_validate(mocker):
-    return mocker.patch.object(ikala, 'validate')
-
-
-@pytest.fixture
-def data_home(tmpdir):
-    return str(tmpdir)
-
-
-@pytest.fixture
-def save_path(data_home):
-    return utils.get_save_path(data_home)
-
-
-@pytest.fixture
-def dataset_path(save_path):
-    return os.path.join(save_path, ikala.DATASET_DIR)
-
-
-def test_validate_invalid(dataset_path, mocker, mock_validator):
-    mock_validator.return_value = (True, True)
-
-    missing_files, invalid_checksums = ikala.validate(dataset_path)
-    assert missing_files and invalid_checksums
-    mock_validator.assert_called_once()
-
-
-def test_validate_valid(dataset_path, mocker, mock_validator):
-    mock_validator.return_value = (False, False)
-
-    missing_files, invalid_checksums = ikala.validate(dataset_path)
-    assert not (missing_files or invalid_checksums)
-    mock_validator.assert_called_once()
+def test_track():
+    data_home = 'tests/resources/mir_datasets/'
+    track = ikala.Track('10161_chorus', data_home=data_home)
+
+    # test attributes are loaded as expected
+    assert track.track_id == '10161_chorus'
+    assert track._data_home == data_home
+    assert track._track_paths == {
+        "audio": [
+            "iKala/Wavfile/10161_chorus.wav",
+            "278ae003cb0d323e99b9a643c0f2eeda"
+        ],
+        "pitch": [
+            "iKala/PitchLabel/10161_chorus.pv",
+            "0d93a011a9e668fd80673049089bbb14"
+        ],
+        "lyrics": [
+            "iKala/Lyrics/10161_chorus.lab",
+            "79bbeb72b422056fd43be4e8d63319ce"
+        ]
+    }
+    assert track.audio_path == 'tests/resources/mir_datasets/iKala/' + \
+        'Wavfile/10161_chorus.wav'
+    assert track.song_id == '10161'
+    assert track.section == 'chorus'
+    assert track.singer_id == '1'
+
+    # test that cached properties don't fail and have the expected type
+    assert type(track.f0) == utils.F0Data
+    assert type(track.lyrics) == utils.LyricData
```

Comment:
On here and line38, I think `is` would be preferred. Such a trivial thing though..